Automattic/wp-calypso
# E2E: Rotate secret

## Proposed Changes

Rotates the store secret we need for e2e tests.

See D128245-code.

## Testing Instructions

Decrypt the file locally and ensure the store secret matches the backend diff.
See PCYsg-vnR-p2 for detailed instructions on how to decrypt.
